packages/apps/virtual-machine/templates/vm.yaml

```
@@ -63,6 +63,7 @@ spec:
             #cloud-config
             ssh_pwauth: {{ if .Values.sshPwauth | default false }}True{{ else }}False{{ end }}
             disable_root: {{ if .Values.disableRoot | default false }}True{{ else }}False{{ end }}
+            user: {{ .Values.user }}
```

Let's make these parameterns optional:

```suggestion
            {{- with .Values.user }}
            user: {{ . }}
            {{- end }}
```

Or we can simple map `cloud-config` as parameter in `values.yaml` as is